### DISCUSSION TITLE
A problem about exodus

### DISCUSSION THREAD

**Discussion by wowodejiajia** [idaholab/moose#22701] [Q&A General] [answered]
Dear MOOSE experts,,
```
[Outputs]
  exodus = flase
  csv = true
  execute_on = 'TIMESTEP_END'
  [./other]
     type = Exodus
     interval = 50
  [../]
[]
```
The .e documentse of each step is too big, so I only want to record 50 steps once. If I put exodus = flase, I will report an error.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You dont need to specify both exodus = true/false and the [other] block of an exodus type.

The syntaxes are not meant to be combined (though they can), just create exodus blocks with the interval you need as you did

Guillaume

**Reply by wowodejiajia** [reply]
I see. Thank you very much
